zeppelin-web/src/app/search/result-list.html

```diff
@@ -12,7 +12,7 @@
 limitations under the License.
 -->
 <div ng-controller="SearchResultCtrl" class="searchResults">
```

Comment:
`ng-controller="SearchResultCtrl"` is not needed here, it is already set in the routes, right now it is instantiating the controller twice